jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -351,12 +351,22 @@ private static void arraycopy(Object[] A1, int offset1, Object[] A2, int offset2
 private static final int FileEncoding = 2;
 private static final int OSEncoding = 3;
 
+/*[IF OpenJ9-RawBuild]*/
+// JCL native
+private static native Properties initProperties(Properties props);
+/*[ENDIF] OpenJ9-RawBuild */
+
 /**
  * If systemProperties is unset, then create a new one based on the values 
  * provided by the virtual machine.
  */
 @SuppressWarnings("nls")
 private static void ensureProperties() {
+/*[IF OpenJ9-RawBuild]*/
+	// invoke JCL native to initialize platform encoding
```

Comment:
Same here - describe why is is required only for raw builds.

Reply:
I think the comment above covers this case as well.